src/wp-admin/includes/ajax-actions.php

```diff
@@ -3012,6 +3012,9 @@ function wp_ajax_query_attachments() {
 		! empty( $_REQUEST['query']['post_status'] ) &&
 		'trash' === $_REQUEST['query']['post_status']
 	) {
+		$posts_query  = new WP_Query();
+		$query_result = $posts_query->query( $query );
+		update_post_parent_caches( $query_result );
```

Comment:
There is no need to run a query here. Just add `update_post_parent_caches( $posts );` here
https://github.com/WordPress/wordpress-develop/blob/6b3a677ce23868c062820f386a3211aca449b275/src/wp-admin/includes/ajax-actions.php#L3043